index.php

```diff
@@ -13,12 +15,27 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
     <body>
-
+    <table>
+        <tr>
+            <th>Brand</th>
+            <th>Name</th>
+            <th>Color</th>
+            <th>Base</th>
+            <th>Quantity Remaining</th>
+            <th>Purchase Date</th>
+        </tr>
         <?php
-        echo '<pre>';
-            var_dump($table);
-        echo '</pre>'
+        foreach ($table as $key => $value) {
+            echo '<tr>';
+            echo "<td>{$value['brand']}</td>";
+            echo "<td>{$value['name']}</td>";
+            echo "<td>{$value['color']}</td>";
+            echo "<td>{$value['base']}</td>";
+            echo "<td>{$value['quantity_left']}</td>";
+            echo "<td>{$value['purchase_date']}</td>";
+            echo '</tr>';
+        }
```

Comment:
Nice work figuring this out, well done! 

I'd now suggest that you move this code into a function that you can call to display your table. Partly because moving code into functions is really good practice, and also because you'll then be able to write a unit test for this new function